stable/spark-history-server/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: spark-history-server
-version: 1.1.2
+version: 1.1.3
```

Comment:
The last number is for bug fixes. This PR is for a new feature, so please make the new version `1.2.0`